docs/converters/tabular_converter.md

```diff
@@ -169,16 +164,16 @@ substitutions:
     own: true
 ```
 The definitions above can be interpreted as:
-  * In all columns that end with `SwitchState` (e.g. `From.Switch State`, `To.Switch State` or just `Switch State`),
-    the word "off" should be replaced with the integer 0 and the word "in" should be replaced with the value 1.
+  * In all columns that end with `SwitchState` (e.g., `From.Switch State`, `To.Switch State` or just `Switch State`),
+    the word "off" should be replaced with integer value 0, and the word "in" integer value 1.
```

Comment:
missing "with" before integer value 1

Also wondering if "in" is correct. I guess that should be "on". Can you check?

Reply:
> missing "with" before integer value 1

Some ellipsis here but could include the preposition 

> Also wondering if "in" is correct. I guess that should be "on". Can you check?

"in" is indeed correct. "on" would be referring to a physical column with a surface